README.md

```diff
@@ -27,18 +27,18 @@ Each tweet will be saved as a JSON file containing the body of the tweet and a l
 
 2. `init.js` - seed the Webtask storage with the first tweet
 
-The script expects a seed tweet at `./tweets/0.json`.
+Run using:
+
+  node init.js /path/to/tweet
```

Comment:
Should the optional parameter be called out like `node init.js [/path/to/tweet]` ?